pubspec.yaml

```diff
@@ -1,17 +1,29 @@
 name: adhara_socket_io
 description: Websocket by socket.io for flutter by adhara, supports both iOS and Android
-version: 1.0.1+1
+version: 1.0.0
+
 homepage: https://github.com/infitio/flutter_socket_io
 
 environment:
-  sdk: ">=2.10.5 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=1.22.6"
 
 dependencies:
   flutter:
     sdk: flutter
   pedantic: ^1.9.2
 
+dev_dependencies:
+  adhara_socket_io_example:
+    path: example
```

Comment:
what is the need for example as a dev dependency?

Reply:
I forgot there reason why, but I have removed it now.